测速失败 https://api.kudian70.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.kudian70.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff789d0>: Failed to resolve 'api.kudian70.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.kudian70.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.kudian70.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78730>: Failed to resolve 'api.kudian70.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.kudian70.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.kudian70.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79480>: Failed to resolve 'api.kudian70.com' ([Errno -2] Name or service not known)"))
测速失败 https://timizy10.cc/api.php/provide/vod/?ac=list: HTTPSConnectionPool(host='timizy10.cc', port=443): Max retries exceeded with url: /api.php/provide/vod/?ac=list (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff79cf0>, 'Connection to timizy10.cc timed out. (connect timeout=5)'))
测速失败 https://timizy10.cc/api.php/provide/vod/?ac=list: HTTPSConnectionPool(host='timizy10.cc', port=443): Max retries exceeded with url: /api.php/provide/vod/?ac=list (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3cd012b5b0>, 'Connection to timizy10.cc timed out. (connect timeout=5)'))
测速失败 https://timizy10.cc/api.php/provide/vod/?ac=list: HTTPSConnectionPool(host='timizy10.cc', port=443): Max retries exceeded with url: /api.php/provide/vod/?ac=list (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff799c0>, 'Connection to timizy10.cc timed out. (connect timeout=5)'))
测速失败 http://llzxcj.com/inc/sck.php?ac=videolist: HTTPConnectionPool(host='llzxcj.com', port=80): Max retries exceeded with url: /inc/sck.php?ac=videolist (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff799f0>: Failed to resolve 'llzxcj.com' ([Errno -2] Name or service not known)"))
测速失败 http://llzxcj.com/inc/sck.php?ac=videolist: HTTPConnectionPool(host='llzxcj.com', port=80): Max retries exceeded with url: /inc/sck.php?ac=videolist (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78820>: Failed to resolve 'llzxcj.com' ([Errno -2] Name or service not known)"))
测速失败 http://llzxcj.com/inc/sck.php?ac=videolist: HTTPConnectionPool(host='llzxcj.com', port=80): Max retries exceeded with url: /inc/sck.php?ac=videolist (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7aa10>: Failed to resolve 'llzxcj.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.feifei67.com/api.php/provide/vod/: HTTPConnectionPool(host='www.feifei67.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3ccff7a6e0>, 'Connection to www.feifei67.com timed out. (connect timeout=5)'))
测速失败 http://llzxcj.com/inc/sck.php?ac=videolist: HTTPConnectionPool(host='llzxcj.com', port=80): Max retries exceeded with url: /inc/sck.php?ac=videolist (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7ac50>: Failed to resolve 'llzxcj.com' ([Errno -2] Name or service not known)"))
测速失败 http://llzxcj.com/inc/sck.php?ac=videolist: HTTPConnectionPool(host='llzxcj.com', port=80): Max retries exceeded with url: /inc/sck.php?ac=videolist (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78160>: Failed to resolve 'llzxcj.com' ([Errno -2] Name or service not known)"))
测速失败 http://llzxcj.com/inc/sck.php?ac=videolist: HTTPConnectionPool(host='llzxcj.com', port=80): Max retries exceeded with url: /inc/sck.php?ac=videolist (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79ed0>: Failed to resolve 'llzxcj.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff79fc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff78160>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff7abc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://sdszyapi.com/home/cjapi/asbb/mc10/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://sdszyapi.com/home/cjapi/asbb/mc10/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://sdszyapi.com/home/cjapi/asbb/mc10/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://www.caiji07.com/home/cjapi/cfcf/mc10/vod/xml: HTTPConnectionPool(host='www.caiji07.com', port=80): Max retries exceeded with url: /home/cjapi/cfcf/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79c30>: Failed to resolve 'www.caiji07.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji07.com/home/cjapi/cfcf/mc10/vod/xml: HTTPConnectionPool(host='www.caiji07.com', port=80): Max retries exceeded with url: /home/cjapi/cfcf/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79a20>: Failed to resolve 'www.caiji07.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji07.com/home/cjapi/cfcf/mc10/vod/xml: HTTPConnectionPool(host='www.caiji07.com', port=80): Max retries exceeded with url: /home/cjapi/cfcf/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7b130>: Failed to resolve 'www.caiji07.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3cd012bee0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff7a170>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff79b40>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.caiji05.com/home/cjapi/cfda/mc10/vod/xml: HTTPConnectionPool(host='www.caiji05.com', port=80): Max retries exceeded with url: /home/cjapi/cfda/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7ae90>: Failed to resolve 'www.caiji05.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji05.com/home/cjapi/cfda/mc10/vod/xml: HTTPConnectionPool(host='www.caiji05.com', port=80): Max retries exceeded with url: /home/cjapi/cfda/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff796c0>: Failed to resolve 'www.caiji05.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji05.com/home/cjapi/cfda/mc10/vod/xml: HTTPConnectionPool(host='www.caiji05.com', port=80): Max retries exceeded with url: /home/cjapi/cfda/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a6e0>: Failed to resolve 'www.caiji05.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji08.com/home/cjapi/cfkl/mc10/vod/xml: HTTPConnectionPool(host='www.caiji08.com', port=80): Max retries exceeded with url: /home/cjapi/cfkl/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78f40>: Failed to resolve 'www.caiji08.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji08.com/home/cjapi/cfkl/mc10/vod/xml: HTTPConnectionPool(host='www.caiji08.com', port=80): Max retries exceeded with url: /home/cjapi/cfkl/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a7a0>: Failed to resolve 'www.caiji08.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji08.com/home/cjapi/cfkl/mc10/vod/xml: HTTPConnectionPool(host='www.caiji08.com', port=80): Max retries exceeded with url: /home/cjapi/cfkl/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79960>: Failed to resolve 'www.caiji08.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji09.com/home/cjapi/cfp0/mc10/vod/xml: HTTPConnectionPool(host='www.caiji09.com', port=80): Max retries exceeded with url: /home/cjapi/cfp0/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79cc0>: Failed to resolve 'www.caiji09.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji09.com/home/cjapi/cfp0/mc10/vod/xml: HTTPConnectionPool(host='www.caiji09.com', port=80): Max retries exceeded with url: /home/cjapi/cfp0/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7b670>: Failed to resolve 'www.caiji09.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji09.com/home/cjapi/cfp0/mc10/vod/xml: HTTPConnectionPool(host='www.caiji09.com', port=80): Max retries exceeded with url: /home/cjapi/cfp0/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3cd012b7c0>: Failed to resolve 'www.caiji09.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji04.com/home/cjapi/cfc7/mc10/vod/xml: HTTPConnectionPool(host='www.caiji04.com', port=80): Max retries exceeded with url: /home/cjapi/cfc7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a140>: Failed to resolve 'www.caiji04.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji04.com/home/cjapi/cfc7/mc10/vod/xml: HTTPConnectionPool(host='www.caiji04.com', port=80): Max retries exceeded with url: /home/cjapi/cfc7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78370>: Failed to resolve 'www.caiji04.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji04.com/home/cjapi/cfc7/mc10/vod/xml: HTTPConnectionPool(host='www.caiji04.com', port=80): Max retries exceeded with url: /home/cjapi/cfc7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7acb0>: Failed to resolve 'www.caiji04.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji03.com/home/cjapi/cfg8/mc10/vod/xml: HTTPConnectionPool(host='www.caiji03.com', port=80): Max retries exceeded with url: /home/cjapi/cfg8/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff787f0>: Failed to resolve 'www.caiji03.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji03.com/home/cjapi/cfg8/mc10/vod/xml: HTTPConnectionPool(host='www.caiji03.com', port=80): Max retries exceeded with url: /home/cjapi/cfg8/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78cd0>: Failed to resolve 'www.caiji03.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji03.com/home/cjapi/cfg8/mc10/vod/xml: HTTPConnectionPool(host='www.caiji03.com', port=80): Max retries exceeded with url: /home/cjapi/cfg8/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff796f0>: Failed to resolve 'www.caiji03.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji10.com/home/cjapi/cfs6/mc10/vod/xml: HTTPConnectionPool(host='www.caiji10.com', port=80): Max retries exceeded with url: /home/cjapi/cfs6/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a590>: Failed to resolve 'www.caiji10.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji10.com/home/cjapi/cfs6/mc10/vod/xml: HTTPConnectionPool(host='www.caiji10.com', port=80): Max retries exceeded with url: /home/cjapi/cfs6/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79d50>: Failed to resolve 'www.caiji10.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji10.com/home/cjapi/cfs6/mc10/vod/xml: HTTPConnectionPool(host='www.caiji10.com', port=80): Max retries exceeded with url: /home/cjapi/cfs6/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79d20>: Failed to resolve 'www.caiji10.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji01.com/home/cjapi/cfd2/mc10/vod/xml: HTTPConnectionPool(host='www.caiji01.com', port=80): Max retries exceeded with url: /home/cjapi/cfd2/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7aa40>: Failed to resolve 'www.caiji01.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji01.com/home/cjapi/cfd2/mc10/vod/xml: HTTPConnectionPool(host='www.caiji01.com', port=80): Max retries exceeded with url: /home/cjapi/cfd2/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3cd0128820>: Failed to resolve 'www.caiji01.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji01.com/home/cjapi/cfd2/mc10/vod/xml: HTTPConnectionPool(host='www.caiji01.com', port=80): Max retries exceeded with url: /home/cjapi/cfd2/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7b580>: Failed to resolve 'www.caiji01.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji02.com/home/cjapi/cfas/mc10/vod/xml: HTTPConnectionPool(host='www.caiji02.com', port=80): Max retries exceeded with url: /home/cjapi/cfas/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a380>: Failed to resolve 'www.caiji02.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji02.com/home/cjapi/cfas/mc10/vod/xml: HTTPConnectionPool(host='www.caiji02.com', port=80): Max retries exceeded with url: /home/cjapi/cfas/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79450>: Failed to resolve 'www.caiji02.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.caiji02.com/home/cjapi/cfas/mc10/vod/xml: HTTPConnectionPool(host='www.caiji02.com', port=80): Max retries exceeded with url: /home/cjapi/cfas/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79420>: Failed to resolve 'www.caiji02.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.dmmapi.com/home/cjapi/asd2c7/mc10/vod/xml: HTTPConnectionPool(host='www.dmmapi.com', port=80): Max retries exceeded with url: /home/cjapi/asd2c7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7bfd0>: Failed to resolve 'www.dmmapi.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.dmmapi.com/home/cjapi/asd2c7/mc10/vod/xml: HTTPConnectionPool(host='www.dmmapi.com', port=80): Max retries exceeded with url: /home/cjapi/asd2c7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78490>: Failed to resolve 'www.dmmapi.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.dmmapi.com/home/cjapi/asd2c7/mc10/vod/xml: HTTPConnectionPool(host='www.dmmapi.com', port=80): Max retries exceeded with url: /home/cjapi/asd2c7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7b5b0>: Failed to resolve 'www.dmmapi.com' ([Errno -2] Name or service not known)"))
测速失败 http://xjjzyapi.com/home/cjapi/askl/mc10/vod/xml: HTTPSConnectionPool(host='xjjzyapi.com', port=443): Max retries exceeded with url: /home/cjapi/askl/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://xjjzyapi.com/home/cjapi/askl/mc10/vod/xml: HTTPSConnectionPool(host='xjjzyapi.com', port=443): Max retries exceeded with url: /home/cjapi/askl/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://xjjzyapi.com/home/cjapi/askl/mc10/vod/xml: HTTPSConnectionPool(host='xjjzyapi.com', port=443): Max retries exceeded with url: /home/cjapi/askl/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://www.888dav.com/api.php/provide/vod/: HTTPSConnectionPool(host='www.888dav.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b550>: Failed to resolve 'www.888dav.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.888dav.com/api.php/provide/vod/: HTTPSConnectionPool(host='www.888dav.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79ab0>: Failed to resolve 'www.888dav.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.888dav.com/api.php/provide/vod/: HTTPSConnectionPool(host='www.888dav.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79ea0>: Failed to resolve 'www.888dav.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b7c0>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a2f0>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7ac80>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://52zyapi.com/home/cjapi/asda/mc10/vod/xml: HTTPSConnectionPool(host='52zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asda/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://52zyapi.com/home/cjapi/asda/mc10/vod/xml: HTTPSConnectionPool(host='52zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asda/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://52zyapi.com/home/cjapi/asda/mc10/vod/xml: HTTPSConnectionPool(host='52zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asda/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc10/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc10/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc10/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc10/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc10/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc10/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://api.apilyzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.apilyzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff79960>, 'Connection to api.apilyzy.com timed out. (connect timeout=5)'))
测速失败 https://api.apilyzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.apilyzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff7bac0>, 'Connection to api.apilyzy.com timed out. (connect timeout=5)'))
测速失败 https://api.apilyzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.apilyzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff7acb0>, 'Connection to api.apilyzy.com timed out. (connect timeout=5)'))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7bbb0>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78820>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a4d0>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.888dav.com/api.php/provide/vod/: HTTPSConnectionPool(host='www.888dav.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79300>: Failed to resolve 'www.888dav.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.888dav.com/api.php/provide/vod/: HTTPSConnectionPool(host='www.888dav.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78340>: Failed to resolve 'www.888dav.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.888dav.com/api.php/provide/vod/: HTTPSConnectionPool(host='www.888dav.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79000>: Failed to resolve 'www.888dav.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.guosezy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.guosezy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b1c0>: Failed to resolve 'api.guosezy.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.guosezy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.guosezy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78610>: Failed to resolve 'api.guosezy.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.guosezy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.guosezy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79000>: Failed to resolve 'api.guosezy.com' ([Errno -2] Name or service not known)"))
测速失败 https://111kkkkk.com/api.php/provide/vod/: HTTPSConnectionPool(host='111kkkkk.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b850>: Failed to resolve '111kkkkk.com' ([Errno -2] Name or service not known)"))
测速失败 https://111kkkkk.com/api.php/provide/vod/: HTTPSConnectionPool(host='111kkkkk.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7bb50>: Failed to resolve '111kkkkk.com' ([Errno -2] Name or service not known)"))
测速失败 https://111kkkkk.com/api.php/provide/vod/: HTTPSConnectionPool(host='111kkkkk.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3cd012b760>: Failed to resolve '111kkkkk.com' ([Errno -2] Name or service not known)"))
测速失败 https://hghhh.com/api.php/provide/vod/: HTTPSConnectionPool(host='hghhh.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b610>: Failed to resolve 'hghhh.com' ([Errno -2] Name or service not known)"))
测速失败 https://hghhh.com/api.php/provide/vod/: HTTPSConnectionPool(host='hghhh.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff797e0>: Failed to resolve 'hghhh.com' ([Errno -2] Name or service not known)"))
测速失败 https://hghhh.com/api.php/provide/vod/: HTTPSConnectionPool(host='hghhh.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78fd0>: Failed to resolve 'hghhh.com' ([Errno -2] Name or service not known)"))
测速失败 http://reba.aishunxs.com/api.php/provide/vod/: HTTPConnectionPool(host='reba.aishunxs.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a8f0>: Failed to resolve 'reba.aishunxs.com' ([Errno -2] Name or service not known)"))
测速失败 http://reba.aishunxs.com/api.php/provide/vod/: HTTPConnectionPool(host='reba.aishunxs.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a560>: Failed to resolve 'reba.aishunxs.com' ([Errno -2] Name or service not known)"))
测速失败 http://reba.aishunxs.com/api.php/provide/vod/: HTTPConnectionPool(host='reba.aishunxs.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3cd012b760>: Failed to resolve 'reba.aishunxs.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.xiuseapi.com/api.php/provide/vod/at/xml: HTTPSConnectionPool(host='api.xiuseapi.com', port=443): Max retries exceeded with url: /api.php/provide/vod/at/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b160>: Failed to resolve 'api.xiuseapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.xiuseapi.com/api.php/provide/vod/at/xml: HTTPSConnectionPool(host='api.xiuseapi.com', port=443): Max retries exceeded with url: /api.php/provide/vod/at/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7ada0>: Failed to resolve 'api.xiuseapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.xiuseapi.com/api.php/provide/vod/at/xml: HTTPSConnectionPool(host='api.xiuseapi.com', port=443): Max retries exceeded with url: /api.php/provide/vod/at/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b070>: Failed to resolve 'api.xiuseapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://sdszyapi.com/home/cjapi/asbb/sea/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/sea/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://sdszyapi.com/home/cjapi/asbb/sea/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/sea/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://sdszyapi.com/home/cjapi/asbb/sea/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/sea/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff7add0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff7abf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff79ff0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://caiji.caomeiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.caomeiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://caiji.caomeiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.caomeiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://caiji.caomeiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.caomeiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 http://www.aikanzyz9.com/inc/apijson_vod.php: HTTPConnectionPool(host='www.aikanzyz9.com', port=80): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79480>: Failed to resolve 'www.aikanzyz9.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.aikanzyz9.com/inc/apijson_vod.php: HTTPConnectionPool(host='www.aikanzyz9.com', port=80): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff79a80>: Failed to resolve 'www.aikanzyz9.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.aikanzyz9.com/inc/apijson_vod.php: HTTPConnectionPool(host='www.aikanzyz9.com', port=80): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3cd012b670>: Failed to resolve 'www.aikanzyz9.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b550>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b6a0>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://caiji.huakuiapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.huakuiapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b400>: Failed to resolve 'caiji.huakuiapi.com' ([Errno -2] Name or service not known)"))
测速失败 http://sdszyapi.com/home/cjapi/asbb/mc10/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://sdszyapi.com/home/cjapi/asbb/mc10/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://sdszyapi.com/home/cjapi/asbb/mc10/vod/xml: HTTPSConnectionPool(host='sdszyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asbb/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xjjzyapi.com/home/cjapi/askl/mc10/vod/xml: HTTPSConnectionPool(host='xjjzyapi.com', port=443): Max retries exceeded with url: /home/cjapi/askl/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xjjzyapi.com/home/cjapi/askl/mc10/vod/xml: HTTPSConnectionPool(host='xjjzyapi.com', port=443): Max retries exceeded with url: /home/cjapi/askl/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xjjzyapi.com/home/cjapi/askl/mc10/vod/xml: HTTPSConnectionPool(host='xjjzyapi.com', port=443): Max retries exceeded with url: /home/cjapi/askl/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://52zyapi.com/home/cjapi/asda/mc10/vod/xml: HTTPSConnectionPool(host='52zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asda/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://52zyapi.com/home/cjapi/asda/mc10/vod/xml: HTTPSConnectionPool(host='52zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asda/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://52zyapi.com/home/cjapi/asda/mc10/vod/xml: HTTPSConnectionPool(host='52zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/asda/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc10/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc10/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://jgczyapi.com/home/cjapi/kld2/mc10/vod/xml: HTTPSConnectionPool(host='jgczyapi.com', port=443): Max retries exceeded with url: /home/cjapi/kld2/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc10/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc10/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 https://xx55zyapi.com/home/cjapi/ascf/mc10/vod/xml: HTTPSConnectionPool(host='xx55zyapi.com', port=443): Max retries exceeded with url: /home/cjapi/ascf/mc10/vod/xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
测速失败 http://www.yrqcaiji.com/home/cjapi/s6c7/mc10/vod/json: HTTPSConnectionPool(host='www.yrqcaiji.com', port=443): Max retries exceeded with url: /home/cjapi/s6c7/mc10/vod/json (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'www.yrqcaiji.com'. (_ssl.c:1007)")))
测速失败 http://www.yrqcaiji.com/home/cjapi/s6c7/mc10/vod/json: HTTPSConnectionPool(host='www.yrqcaiji.com', port=443): Max retries exceeded with url: /home/cjapi/s6c7/mc10/vod/json (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'www.yrqcaiji.com'. (_ssl.c:1007)")))
测速失败 http://www.yrqcaiji.com/home/cjapi/s6c7/mc10/vod/json: HTTPSConnectionPool(host='www.yrqcaiji.com', port=443): Max retries exceeded with url: /home/cjapi/s6c7/mc10/vod/json (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'www.yrqcaiji.com'. (_ssl.c:1007)")))
测速失败 http://xcncaiji.com/home/cjapi/s6bb/mc10/vod/json: HTTPSConnectionPool(host='xcncaiji.com', port=443): Max retries exceeded with url: /home/cjapi/s6bb/mc10/vod/json (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'xcncaiji.com'. (_ssl.c:1007)")))
测速失败 http://xcncaiji.com/home/cjapi/s6bb/mc10/vod/json: HTTPSConnectionPool(host='xcncaiji.com', port=443): Max retries exceeded with url: /home/cjapi/s6bb/mc10/vod/json (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'xcncaiji.com'. (_ssl.c:1007)")))
测速失败 http://xcncaiji.com/home/cjapi/s6bb/mc10/vod/json: HTTPSConnectionPool(host='xcncaiji.com', port=443): Max retries exceeded with url: /home/cjapi/s6bb/mc10/vod/json (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'xcncaiji.com'. (_ssl.c:1007)")))
测速失败 https://www.dmmapi.com/home/cjapi/asd2c7/mc10/vod/xml: HTTPSConnectionPool(host='www.dmmapi.com', port=443): Max retries exceeded with url: /home/cjapi/asd2c7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79cc0>: Failed to resolve 'www.dmmapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.dmmapi.com/home/cjapi/asd2c7/mc10/vod/xml: HTTPSConnectionPool(host='www.dmmapi.com', port=443): Max retries exceeded with url: /home/cjapi/asd2c7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3cd012b6d0>: Failed to resolve 'www.dmmapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.dmmapi.com/home/cjapi/asd2c7/mc10/vod/xml: HTTPSConnectionPool(host='www.dmmapi.com', port=443): Max retries exceeded with url: /home/cjapi/asd2c7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7abf0>: Failed to resolve 'www.dmmapi.com' ([Errno -2] Name or service not known)"))
测速失败 https://www.caiji10.com/home/cjapi/cfs6/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji10.com', port=443): Max retries exceeded with url: /home/cjapi/cfs6/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79930>: Failed to resolve 'www.caiji10.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji10.com/home/cjapi/cfs6/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji10.com', port=443): Max retries exceeded with url: /home/cjapi/cfs6/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b490>: Failed to resolve 'www.caiji10.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji10.com/home/cjapi/cfs6/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji10.com', port=443): Max retries exceeded with url: /home/cjapi/cfs6/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b250>: Failed to resolve 'www.caiji10.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji09.com/home/cjapi/cfp0/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji09.com', port=443): Max retries exceeded with url: /home/cjapi/cfp0/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b130>: Failed to resolve 'www.caiji09.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji09.com/home/cjapi/cfp0/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji09.com', port=443): Max retries exceeded with url: /home/cjapi/cfp0/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3cd012b6d0>: Failed to resolve 'www.caiji09.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji09.com/home/cjapi/cfp0/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji09.com', port=443): Max retries exceeded with url: /home/cjapi/cfp0/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b310>: Failed to resolve 'www.caiji09.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji08.com/home/cjapi/cfkl/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji08.com', port=443): Max retries exceeded with url: /home/cjapi/cfkl/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7abc0>: Failed to resolve 'www.caiji08.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji08.com/home/cjapi/cfkl/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji08.com', port=443): Max retries exceeded with url: /home/cjapi/cfkl/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7ba00>: Failed to resolve 'www.caiji08.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji08.com/home/cjapi/cfkl/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji08.com', port=443): Max retries exceeded with url: /home/cjapi/cfkl/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b3d0>: Failed to resolve 'www.caiji08.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji07.com/home/cjapi/cfcf/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji07.com', port=443): Max retries exceeded with url: /home/cjapi/cfcf/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b760>: Failed to resolve 'www.caiji07.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji07.com/home/cjapi/cfcf/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji07.com', port=443): Max retries exceeded with url: /home/cjapi/cfcf/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79150>: Failed to resolve 'www.caiji07.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji07.com/home/cjapi/cfcf/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji07.com', port=443): Max retries exceeded with url: /home/cjapi/cfcf/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78460>: Failed to resolve 'www.caiji07.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji06.com/home/cjapi/cfbb/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji06.com', port=443): Max retries exceeded with url: /home/cjapi/cfbb/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7ada0>: Failed to resolve 'www.caiji06.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji06.com/home/cjapi/cfbb/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji06.com', port=443): Max retries exceeded with url: /home/cjapi/cfbb/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7aef0>: Failed to resolve 'www.caiji06.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji06.com/home/cjapi/cfbb/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji06.com', port=443): Max retries exceeded with url: /home/cjapi/cfbb/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78a00>: Failed to resolve 'www.caiji06.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji05.com/home/cjapi/cfda/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji05.com', port=443): Max retries exceeded with url: /home/cjapi/cfda/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff798d0>: Failed to resolve 'www.caiji05.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji05.com/home/cjapi/cfda/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji05.com', port=443): Max retries exceeded with url: /home/cjapi/cfda/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7bc40>: Failed to resolve 'www.caiji05.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji05.com/home/cjapi/cfda/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji05.com', port=443): Max retries exceeded with url: /home/cjapi/cfda/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7b520>: Failed to resolve 'www.caiji05.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji04.com/home/cjapi/cfc7/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji04.com', port=443): Max retries exceeded with url: /home/cjapi/cfc7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79d50>: Failed to resolve 'www.caiji04.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji04.com/home/cjapi/cfc7/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji04.com', port=443): Max retries exceeded with url: /home/cjapi/cfc7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79ea0>: Failed to resolve 'www.caiji04.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji04.com/home/cjapi/cfc7/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji04.com', port=443): Max retries exceeded with url: /home/cjapi/cfc7/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78130>: Failed to resolve 'www.caiji04.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji03.com/home/cjapi/cfg8/zp/vod/xml: HTTPSConnectionPool(host='www.caiji03.com', port=443): Max retries exceeded with url: /home/cjapi/cfg8/zp/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3cd012b700>: Failed to resolve 'www.caiji03.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji03.com/home/cjapi/cfg8/zp/vod/xml: HTTPSConnectionPool(host='www.caiji03.com', port=443): Max retries exceeded with url: /home/cjapi/cfg8/zp/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff799c0>: Failed to resolve 'www.caiji03.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji03.com/home/cjapi/cfg8/zp/vod/xml: HTTPSConnectionPool(host='www.caiji03.com', port=443): Max retries exceeded with url: /home/cjapi/cfg8/zp/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79e70>: Failed to resolve 'www.caiji03.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji02.com/home/cjapi/cfas/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji02.com', port=443): Max retries exceeded with url: /home/cjapi/cfas/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78580>: Failed to resolve 'www.caiji02.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji02.com/home/cjapi/cfas/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji02.com', port=443): Max retries exceeded with url: /home/cjapi/cfas/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a080>: Failed to resolve 'www.caiji02.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji02.com/home/cjapi/cfas/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji02.com', port=443): Max retries exceeded with url: /home/cjapi/cfas/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78b50>: Failed to resolve 'www.caiji02.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji01.com/home/cjapi/cfd2/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji01.com', port=443): Max retries exceeded with url: /home/cjapi/cfd2/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3cd012b700>: Failed to resolve 'www.caiji01.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji01.com/home/cjapi/cfd2/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji01.com', port=443): Max retries exceeded with url: /home/cjapi/cfd2/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78a00>: Failed to resolve 'www.caiji01.com' ([Errno -5] No address associated with hostname)"))
测速失败 https://www.caiji01.com/home/cjapi/cfd2/mc10/vod/xml: HTTPSConnectionPool(host='www.caiji01.com', port=443): Max retries exceeded with url: /home/cjapi/cfd2/mc10/vod/xml (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a380>: Failed to resolve 'www.caiji01.com' ([Errno -5] No address associated with hostname)"))
测速失败 http://www.987caiji.com/api/max.php: HTTPConnectionPool(host='www.987caiji.com', port=80): Max retries exceeded with url: /api/max.php (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7aaa0>: Failed to resolve 'www.987caiji.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.987caiji.com/api/max.php: HTTPConnectionPool(host='www.987caiji.com', port=80): Max retries exceeded with url: /api/max.php (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a980>: Failed to resolve 'www.987caiji.com' ([Errno -2] Name or service not known)"))
测速失败 http://www.987caiji.com/api/max.php: HTTPConnectionPool(host='www.987caiji.com', port=80): Max retries exceeded with url: /api/max.php (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78d00>: Failed to resolve 'www.987caiji.com' ([Errno -2] Name or service not known)"))
测速失败 http://99zyba.com/api.php/provide/vod/: HTTPConnectionPool(host='99zyba.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff78160>: Failed to resolve '99zyba.com' ([Errno -2] Name or service not known)"))
测速失败 http://99zyba.com/api.php/provide/vod/: HTTPConnectionPool(host='99zyba.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7a2f0>: Failed to resolve '99zyba.com' ([Errno -2] Name or service not known)"))
测速失败 http://99zyba.com/api.php/provide/vod/: HTTPConnectionPool(host='99zyba.com', port=80): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3ccff7b910>: Failed to resolve '99zyba.com' ([Errno -2] Name or service not known)"))
测速失败 https://kudouzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='kudouzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://kudouzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='kudouzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://kudouzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='kudouzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://api.kdapi.info/api.php/provide/vod/: HTTPSConnectionPool(host='api.kdapi.info', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79360>: Failed to resolve 'api.kdapi.info' ([Errno -2] Name or service not known)"))
测速失败 https://api.kdapi.info/api.php/provide/vod/: HTTPSConnectionPool(host='api.kdapi.info', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff794b0>: Failed to resolve 'api.kdapi.info' ([Errno -2] Name or service not known)"))
测速失败 https://api.kdapi.info/api.php/provide/vod/: HTTPSConnectionPool(host='api.kdapi.info', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a590>: Failed to resolve 'api.kdapi.info' ([Errno -2] Name or service not known)"))
测速失败 https://api.kudian70.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.kudian70.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a950>: Failed to resolve 'api.kudian70.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.kudian70.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.kudian70.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff79ab0>: Failed to resolve 'api.kudian70.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.kudian70.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.kudian70.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f3ccff78d00>: Failed to resolve 'api.kudian70.com' ([Errno -2] Name or service not known)"))
测速失败 https://api.ykapi.net/api.php/provide/vod/: HTTPSConnectionPool(host='api.ykapi.net', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ccff7bf40>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://api.ykapi.net/api.php/provide/vod/: HTTPSConnectionPool(host='api.ykapi.net', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ccff7be50>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://api.ykapi.net/api.php/provide/vod/: HTTPSConnectionPool(host='api.ykapi.net', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ccff7baf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://caiji.naichaapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.naichaapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ccff78a90>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://caiji.naichaapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.naichaapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ccff7ba30>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://caiji.naichaapi.com/inc/apijson_vod.php: HTTPSConnectionPool(host='caiji.naichaapi.com', port=443): Max retries exceeded with url: /inc/apijson_vod.php (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ccff78cd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 https://cj.apiabzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='cj.apiabzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://cj.apiabzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='cj.apiabzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://cj.apiabzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='cj.apiabzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://api.apilyzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.apilyzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff7a830>, 'Connection to api.apilyzy.com timed out. (connect timeout=5)'))
测速失败 https://api.apilyzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.apilyzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff78ac0>, 'Connection to api.apilyzy.com timed out. (connect timeout=5)'))
测速失败 https://api.apilyzy.com/api.php/provide/vod/: HTTPSConnectionPool(host='api.apilyzy.com', port=443): Max retries exceeded with url: /api.php/provide/vod/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ccff7aef0>, 'Connection to api.apilyzy.com timed out. (connect timeout=5)'))
测速失败 https://bttcj.com/inc/sapi.php: HTTPSConnectionPool(host='bttcj.com', port=443): Max retries exceeded with url: /inc/sapi.php (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'bttcj.com'. (_ssl.c:1007)")))
测速失败 https://bttcj.com/inc/sapi.php: HTTPSConnectionPool(host='bttcj.com', port=443): Max retries exceeded with url: /inc/sapi.php (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'bttcj.com'. (_ssl.c:1007)")))
测速失败 https://bttcj.com/inc/sapi.php: HTTPSConnectionPool(host='bttcj.com', port=443): Max retries exceeded with url: /inc/sapi.php (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'bttcj.com'. (_ssl.c:1007)")))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 https://www.afasu.com/api/xml.php: HTTPSConnectionPool(host='www.afasu.com', port=443): Max retries exceeded with url: /api/xml.php (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff79720>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff78340>: Failed to establish a new connection: [Errno 111] Connection refused'))
测速失败 http://www.ggmmzy.com:9999/inc/xml: HTTPConnectionPool(host='www.ggmmzy.com', port=9999): Max retries exceeded with url: /inc/xml (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ccff78520>: Failed to establish a new connection: [Errno 111] Connection refused'))
